app/views/curation_concerns/base/_member_of_collections.html.erb

```diff
@@ -0,0 +1,12 @@
+    <% if defined?(presenter.member_of_collection_presenters) && presenter.member_of_collection_presenters.size > 0 %>
```

Comment:
Why do we need to check for `defined?(presenter.member_of_collection_presenters)`?